wiki/osu!_Program_Files/en.md

```diff
@@ -1,113 +1,117 @@
----
-needs_cleanup: true  # needs to be rewritten (issue #4753)
----
-
 # osu! program files
 
-![osu! Program Files in a nutshell](img/osu!-program-files.png "osu! Program Files in a nutshell")
+![The file structure of osu!'s installation folder, on Windows and macOS](img/file_structure.jpg "The file structure of osu!'s installation folder, on Windows and macOS")
 
-## Location
+The **osu! program files** are a set of files that run osu! and keep track of different activities as user's play.
 
-### Windows
+## Installation paths
 
-| Current default filepath | Old installer filepath |
-| :-- | :-- |
-| `C:\Users\<Username>\AppData\Local\osu!` | `C:\Program Files\osu!` / `C:\Program Files(x86)\osu!` (for 7 64-bit or 8) |
+osu! is installed in the following locations, by default:
```

Comment:
```suggestion
By default, osu! is installed in the following locations:
```
still not too hot on the use of a table here

Reply:
Fixed in 18b1c34